### DISCUSSION TITLE
es_api_token

### DISCUSSION THREAD

**Discussion by khangcnttspkt** [jertel/elastalert2#409] [Q&A] [answered]
Hi everyone,

How i using "es_api_token" options

Tks.

**Comment by nsano-rururu** [comment, marked as answer]
I searched the source code but found no es_api_token. Perhaps you forgot to erase the description.

**Reply by nsano-rururu** [reply]
@jertel

Do you know anything about es_api_token?

**Reply by nsano-rururu** [reply]
@jertel 

Did you mistakenly write es_api_token when you tried to write es_api_key?
https://github.com/jertel/elastalert2/commit/d6862f1da6289e0e296a73d2d86cc2b9dd87172b#diff-118901b49d04fe09c014bd8fcf4d7fefa57ee2f4e42e2c1dadaf74b9fc560564

**Reply by jertel** [reply]
That was added by @vbisserie in this PR: https://github.com/jertel/elastalert2/pull/208/files. It does appear to be a mistake.

**Reply by nsano-rururu** [reply]
@khangcnttspkt 

See the pull request below and the "es_api_key" in the documentation.
https://github.com/jertel/elastalert2/pull/208
